### ISSUE TITLE
Typescript Types not available in npm package

### ISSUE THREAD

**Issue by ChromeQ** [rollbar/rollbar-react#31]
Looks like some types were included in [v0.9.0](https://github.com/rollbar/rollbar-react/releases/tag/v0.9.0) but when I install via `npm install @rollbar/react` then the correct version is installed but types are still missing.

I think the index.d.ts is not included in the package:
![image](https://user-images.githubusercontent.com/625203/153798919-152e53dd-60d8-4b71-86a5-b550a7b25c66.png)

So the types are not available in the code:
```sh
error TS7016: Could not find a declaration file for module '@rollbar/react'. '.../node_modules/@rollbar/react/lib/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/rollbar__react` if it exists or add a new declaration (.d.ts) file containing `declare module '@rollbar/react';`

8 import { Provider as RollbarProvider, ErrorBoundary as RollbarErrorBoundary } from '@rollbar/react';
```

**Comment by ChromeQ** [comment]
Thanks for fixing this @waltjones , will there be another release soon as latest version is still 0.9.0?

**Comment by waltjones** [comment]
This is now published in v0.10.0. https://github.com/rollbar/rollbar-react/releases/tag/v0.10.0
